src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import core.basesyntax.Figure;
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(15)];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = new FigureSupplier().getFigure();
```

Comment:
new FigureSupplier is created on each loop iteration - fix this

Reply:
Ksenia, we should create an array of shapes, but how in another way to create a new figure if not with new iteration ?